### ISSUE TITLE
Change user ID

### ISSUE THREAD

**Issue by Tepepaz** [yairm210/Unciv#2360]
Hi there and first of all thank you for all your work behind this game. 

I'm playing a multilayer game with a friend. 
Due to an issue with his device he had to uninstall and re-install the game. 
When he tries to re-join the game we were playing he is assigned to a random non-existent player's spot instead of the original civilization he played since the beginning.

I guess this happens because his user ID changed when he reinstalled the game. 
Now, to fix this I could either update his old ID in the multilayer game I created or he could change his user ID with the old one. 

Either way, I didn't find any way to update the user ID in the game. 

Suggestions?

**Comment by yairm210** [comment]
In the end this is more an issue of trust that technology.
The big question being, "do we want to let everyone set their own ID"? 
If yes, the people could set themselves to b other people, and could ruin things or other people.
On the other hand, the current playerbase is not that bug, and I think the advantages for now outweigh the disadvantages

**Comment by Tepepaz** [comment]
Thank you for your reply.

I totally see your point. That's why I also suggested to let people update the users IDs in the multilayer game they created thus they admin. But I guess thus would be more a feature suggestion than a bug.

**Comment by rh-github-2015** [comment]
This ties in with an old request to be able to export/import - such a feature would carry ID over. I played with that and got it even half-working - see [this branch](https://github.com/rh-github-2015/Unciv/tree/Import-Export-UI). 

Problem with that - old Android permissions model, it works nicely on Lineage 16 (which is the ROM on my hardware testing base, based on AOSP 9.0 Pie) after manually granting permission but not at all on the current emulator. Getting the proper way to ask for permissions in - or doing it via share api - is beyond me ATM, and our base LibGdx doesn't help as far as I could see. Therefore I abandoned the branch.

That said there's no tamper protection, so anyone with root and a text editor could impersonate any ID.... Should we add tamper protection? Not worth it - she who roots would just as well look in our sources to cheat that too.
